subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/OutgoingVariantsReportTask.java

```diff
@@ -91,12 +92,12 @@ void buildReport() {
             reportNoMatchingVariant(configurations, output);
         } else {
             Legend legend = new Legend();
-            configurations.forEach(cnf -> reportVariant((ConfigurationInternal) cnf, output, legend));
+            configurations.forEach(cnf -> reportVariant((ConfigurationInternal) cnf,new ProjectBackedModule(getProject()), output, legend));
```

Comment:
I think there is a whitespace missing here.

Reply:
Yes :-(